tools/fpgabist/fpgabist

```diff
@@ -39,6 +39,7 @@ FEATURES = ['fme', 'port', 'temp', 'power', 'errors all']
 
 def main(args):
     bdf = bist_common.get_all_fpga_bdfs()
+    cwd = os.path.dirname(os.path.realpath(__file__))
```

Comment:
again you should renamed `cwd`.  Also it might be better to get this value from bist_common.  That way if it is ever changed you only need to make the update in one place.